cmake/remillConfig.cmake.in

```diff
@@ -12,79 +12,60 @@
-  
-  add_library(remill_os STATIC IMPORTED)
-  set_property(TARGET remill_os PROPERTY IMPORTED_LOCATION "@REMILL_OS_LIBRARY_LOCATION@")
-  
-  add_library(remill_arch STATIC IMPORTED)
-  set_property(TARGET remill_arch PROPERTY IMPORTED_LOCATION "@REMILL_ARCH_LIBRARY_LOCATION@")
-  
-  add_library(remill_arch_x86 STATIC IMPORTED)
-  set_property(TARGET remill_arch_x86 PROPERTY IMPORTED_LOCATION "@REMILL_ARCH_X86_LIBRARY_LOCATION@")
-  
-  add_library(remill_arch_aarch32 STATIC IMPORTED)
-  set_property(TARGET remill_arch_aarch32 PROPERTY IMPORTED_LOCATION "@REMILL_ARCH_AARCH32_LIBRARY_LOCATION@")
-  
-  add_library(remill_arch_aarch64 STATIC IMPORTED)
-  set_property(TARGET remill_arch_aarch64 PROPERTY IMPORTED_LOCATION "@REMILL_ARCH_AARCH64_LIBRARY_LOCATION@")
-  
-  add_library(remill_arch_sparc32 STATIC IMPORTED)
-  set_property(TARGET remill_arch_sparc32 PROPERTY IMPORTED_LOCATION "@REMILL_ARCH_SPARC32_LIBRARY_LOCATION@")
-  
-  add_library(remill_arch_sparc64 STATIC IMPORTED)
-  set_property(TARGET remill_arch_sparc64 PROPERTY IMPORTED_LOCATION "@REMILL_ARCH_SPARC64_LIBRARY_LOCATION@")
-  
-  add_library(remill_version STATIC IMPORTED)
-  set_property(TARGET remill_version PROPERTY IMPORTED_LOCATION "@REMILL_VERSION_LIBRARY_LOCATION@")
-  
-  add_library(remill INTERFACE)
-  target_link_libraries(remill INTERFACE
-    ${LINKER_START_GROUP}
-    remill_bc
-    remill_os
-    remill_arch
-    remill_arch_x86
-    remill_arch_aarch32
-    remill_arch_aarch64
-    remill_arch_sparc32
-    remill_arch_sparc64
-    remill_version
-    @THIRDPARTY_LIBRARY_FILES@
-    ${LINKER_END_GROUP}
-  )
-  
-  target_include_directories(remill INTERFACE @THIRDPARTY_INCLUDE_DIRECTORIES@)
-  target_include_directories(remill INTERFACE @REMILL_INCLUDE_LOCATION@)
-  target_compile_options(remill INTERFACE @REMILL_COMPILE_OPTIONS@)
-  target_compile_definitions(remill INTERFACE @REMILL_COMPILE_DEFINITIONS@)
 
-  # Add a dummy 'semantics' target to satisfy the protobuf generator
-  add_custom_target(semantics)
+  # External libs
+  include(CMakeFindDependencyMacro)
+  find_dependency(XED)
+  find_dependency(glog)
+  find_dependency(LLVM)
+  # NOTE: If changing this, also replicate in CMakeLists.txt
+  if (LLVM_WITH_Z3)
+    find_dependency(Z3)
+    get_target_property(LLVMSupport_LIBS LLVMSupport INTERFACE_LINK_LIBRARIES)
+    list(REMOVE_ITEM LLVMSupport_LIBS Z3)
+    list(APPEND LLVMSupport_LIBS z3::libz3)
+    set_target_properties(LLVMSupport PROPERTIES
+      INTERFACE_LINK_LIBRARIES "${LLVMSupport_LIBS}")
+  endif()
+
+  # Exported Targets
+  include("${CMAKE_CURRENT_LIST_DIR}/remillTargets.cmake")
+
+  if(TARGET thirdparty_llvm)
+    # Need this so that LLVM doesn't hardcode it's include directories
+    set_target_properties(thirdparty_llvm PROPERTIES
+      INTERFACE_INCLUDE_DIRECTORIES "${LLVM_INCLUDE_DIRS}"
+      INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${LLVM_INCLUDE_DIRS}"
+      )
+  endif()
```

Comment:
This is the actual change to the file as moved from `cmake/remillConfig_vcpkg.cmake.in -> cmake/remillConfig.cmake.in`